SECURITY.md

```diff
@@ -7,15 +7,4 @@ released tagged minor version. All previous versions are unsupported.
 
 ## Reporting a Vulnerability
 
-Users are able to open helpdesk tickets on [Discourse](https://discuss.dockstore.org/). Users can create helpdesk tickets in case of privacy complaints, security vulnerabilities, or any other urgent matter related to Dockstore. Helpdesk tickets will be addressed by Dockstore administrators.
-
-The following steps can be taken to create a helpdesk ticket (also shown [here](https://discuss.dockstore.org/t/opening-helpdesk-tickets/1506)).
-
-1. Navigate to [Discourse](https://discuss.dockstore.org/) and login.
-2. Select your profile icon, located in the top right corner of the screen.
-3. Select the `mail` icon, located in the dropdown.
-4. Send a message to the `dockstore_admins` group.
-
-Note
-
-> If you are unable to see a New Message button on the mail page, you may be considered a new user and have insufficient privileges. Entering 5 topics and viewing 30 posts over a minimum of 10 minutes will raise your privileges. You will be notified of any privilege changes to your account via the mailbox.
+Use the "Report a vulnerability" button to the upper right to report security vulnerabilities. A helpdesk ticket on [discuss.dockstore.org](https://discuss.dockstore.org/t/opening-helpdesk-tickets/1506) may be an option for privacy complaints or other urgent non-security matters. Both options will be addressed by Dockstore administrators. 
```

Comment:
Are we expecting users to report vulnerabilities to the specific repo it affects or should they all go to dockstore/dockstore like GitHub issues? 

For notifications, how will we know when there's a new vulnerability reported?

Reply:
> Are we expecting users to report vulnerabilities to the specific repo it affects or should they all go to dockstore/dockstore like GitHub issues? 

I think all repos have the ability to receive reports right now. 
We could limit things, but I think that's ok for now. The UI and/or discourse refer to the main repo. 

> For notifications, how will we know when there's a new vulnerability reported? 

I think it's the normal notification mechanism like with PRs

Reply:
Yup, regular notifications https://github.com/dockstore/dockstore/security/advisories/GHSA-fp43-4wjf-48r4